docs/concepts/EvictionAndDisruptionBudget/README.md

```diff
@@ -0,0 +1,47 @@
+# Eviction & Placement Disruption Budget
+
+This document explains the concept of `Eviction` and `Placement Disruption Budget` in the context of the fleet.
+
+## Overview
+
+`Eviction` pertains to the act of removing resources from a target cluster propagated by a resource placement object from the hub cluster.
+
+The `Placement Disruption Budget` object protects against voluntary disruption, and in the case of the fleet, the only allowed voluntary disruption as of now is eviction.
```

Comment:
Hi Arvind! I think readers might need some additional info. on voluntary disruption.

Reply:
Or we could list the cases that DB can (and cannot) protect against explicitly.

Reply:
Same goes for other refs about voluntary disruptions in the text.